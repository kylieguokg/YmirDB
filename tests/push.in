push b 23929
set jam 10 20 304 55
get jam
push jam -10000
get jam
pop jam
pop jam
pop jam
pop jam
pop jam
push jam 0
push jam
set jelly 29
push jelly jam
list entries
bye
