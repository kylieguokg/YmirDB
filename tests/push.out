> no such key

> ok

> [10 20 304 55]

> ok

> [-10000 10 20 304 55]

> -10000

> 10

> 20

> 304

> 55

> ok

> ok

> ok

> ok

> jelly [jam 29]
jam [0]

> bye
